cmd/bedFormat/bedFormat.go

```diff
@@ -6,23 +6,40 @@ import (
 	"flag"
 	"fmt"
 	"github.com/vertgenlab/gonomics/bed"
+	"github.com/vertgenlab/gonomics/chromInfo"
 	"github.com/vertgenlab/gonomics/common"
 	"github.com/vertgenlab/gonomics/convert"
 	"github.com/vertgenlab/gonomics/exception"
 	"github.com/vertgenlab/gonomics/fileio"
+	"github.com/vertgenlab/gonomics/numbers"
 	"log"
 )
 
 func bedFormat(s Settings) {
 	var err error
+	var inMap bool
+	var sizes map[string]chromInfo.ChromInfo
 	ch := bed.GoReadToChan(s.InFile)
 	out := fileio.EasyCreate(s.OutFile)
 
 	if s.EnsemblToUCSC && s.UCSCToEnsembl {
 		log.Fatalf("Both conversions (UCSCToEnsembl and EnsemblToUCSC) are incompatable.")
 	}
+	if s.ChromSizeFile == "" && s.PadLength > 0 {
+		log.Fatalf("Must specify a chromFile to use the padLength option.")
+	}
+	if s.ChromSizeFile != "" && s.PadLength > 0 {
+		sizes = chromInfo.ReadToMap(s.ChromSizeFile)
+	}
 
 	for v := range ch {
+		if s.PadLength > 0 {
+			if _, inMap = sizes[v.Chrom]; !inMap {
```

Comment:
Not a big deal, but we are throwing away the value on this line with the underscore, but then a couple lines later we are using the value to set v.ChromEnd.  We could have as single lookup by saving the value on this line.